db/...ma.rb

```diff
@@ -0,0 +1,43 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
what is this file?

Reply:
something that autogenrated when i was playing around with migrations on heroku, anyway it's gone now